src/styles/block/nav.scss

```diff
@@ -0,0 +1,27 @@
+.nav {
+  &__list {
+    display: flex;
+    list-style: none;
+
+    margin: 0;
+    padding: 0;
+  }
+
+  &__link {
+    text-transform: uppercase;
+    text-decoration: none;
+    transition-duration: $transition-speed;
+    line-height: $header-height;
+
+    display: block;
+    color: $color-text;
+  }
+
+  &__item:not(:last-child) {
+    padding-right: 20px;
```

Comment:
It's recommended to use variables for spacing values like padding and margin to maintain consistency across the project. Consider replacing the `20px` value with a variable from your `variables.scss` file if applicable.